maths/factorial_iterative.py

```diff
@@ -0,0 +1,19 @@
+# factorial of a number
+
+
+n=int(input("Enter a number"))
```

Comment:
This line will fail our automated testing unless it is indented under `if __name__ == "__main__":` as discussed in CONTRIBUTING.md